pdq/abstractness.go

```diff
@@ -0,0 +1,19 @@
+package pdq
+
+import (
+	"fmt"
+)
+
+func CalcAbstractness(abstractElementsNum int, concreteElementsNum int) (float64, error) {
+	if abstractElementsNum < 0 || concreteElementsNum < 0 {
+		return -1, fmt.Errorf("abstractElementsNum %v, concreteElementsNum %v; values must be >= 0", abstractElementsNum, concreteElementsNum)
```

Comment:
returning -1 on err, which can be confusing (since -1 can be interpreted as a valid abstraction value), returning 0, which is more standard practice when returning an err